pkg/platform/github.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package github provides the functionality to send requests to the GitHub API.
+package platform
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/google/go-github/v53/github"
+	"golang.org/x/oauth2"
+
+	gh "github.com/abcxyz/guardian/pkg/github"
```

Comment:
I prefer keeping "local" package un-aliased, i dont know if this is idiomatic or not but /shrug